File: Easy/[27] Remove Element/Java Solution/[27] Remove Element.java
/*
Accepted
27 [Easy]
Runtime: 0 ms, faster than 100.00% of Java online submissions for Remove Element.
Memory Usage: 42.52 MB, less than 5.78% of Java online submissions for Remove Element.
 */
class Solution {
    public int removeElement(int[] nums, int val) {
        int count=0,a[]=new int[nums.length];
        for(int i=0;i<nums.length;i++) {
            if(nums[i]==val)
                nums[i]=200;
        }
        for(int i=0;i<nums.length;i++) {
            if(nums[i]!=200) {
                a[count]=nums[i];
                count++;
            }
        }
        for(int i=count;i<nums.length;i++)
            a[i]=200;
        for(int i=0;i<nums.length;i++) 
            nums[i]=a[i];
        return count;
    }
}

File: Easy/[27] Remove Element/Python3 Solution/[27] Remove Element.py
"""
Accepted
27 [Easy]
Runtime: 33 ms, faster than 90.96% of Python3 online submissions for Remove Element.
Memory Usage: 17.43 MB, less than 14.56% of Python3 online submissions for Remove Element.
"""
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        k = 0
        for i in range(len(nums)):
            while nums[i] == val:
                del nums[i]
                nums.append(-1)
                k += 1

        return len(nums) - k